bot_local.py

```diff
@@ -13,12 +13,14 @@
 
 class ConsoleBot(Bot):
     """Fake console bot"""
-    async def say(self, *, channel_id, text='', attachments=None, message_type=''):  # pylint: disable=unused-argument
+    async def say(
+            self, *, channel_id, text='', attachments=None, message_type='', is_announcement=False
+    ):  # pylint: disable=unused-argument
         """Print messages to stdout"""
         attachment_text = ''
         if attachments is not None:
             attachment_text = attachments[0].get('text', '')
-        line = " ".join(word for word in [text, attachment_text, message_type] if word)
+        line = " ".join(word for word in [text, attachment_text, message_type, is_announcement] if word)
```

Comment:
`line = " ".join(word for word in [text, attachment_text, message_type, str(is_announcement)] if word)`

Otherwise this error occurs:
```
Traceback (most recent call last):
  File "/Users/mbertrand/code/release-script/bot.py", line 931, in handle_message
    loop=loop
  File "/Users/mbertrand/code/release-script/bot.py", line 903, in run_command
    manager=manager,
  File "/Users/mbertrand/code/release-script/bot.py", line 570, in commits_since_last_release
    is_announcement=True,
  File "/Users/mbertrand/code/release-script/bot.py", line 249, in say_with_attachment
    is_announcement=is_announcement,
  File "bot_local.py", line 23, in say
    line = " ".join(word for word in [text, attachment_text, message_type, is_announcement] if word)
TypeError: sequence item 2: expected str instance, bool found
```

Reply:
:man_facepalming:

Reply:
Fixed and added some tests